frontend/src/components/MeetingRoom/LayoutButton/LayoutButton.spec.tsx

```diff
@@ -54,11 +54,21 @@ describe('LayoutButton', () => {
 
   it('should render the tooltip title that mentions switching layouts is not allowed when screenshare is present if currently in the active speaker mode', async () => {
     (useSessionContext as Mock).mockReturnValue({ layoutMode: 'active-speaker' });
-    render(<LayoutButton isScreenSharePresent />);
+    render(<LayoutButton isScreenSharePresent isPinnedPresent={false} />);
     const button = await screen.findByRole('button');
     await userEvent.hover(button);
     const tooltip = await screen.findByRole('tooltip');
     expect(tooltip).toBeInTheDocument();
     expect(tooltip.textContent).toBe('Cannot switch layout while screen share is active');
   });
+
+  it('should render the tooltip title that mentions switching layouts is not allowed when a pinned participant is present', async () => {
+    (useSessionContext as Mock).mockReturnValue({ layoutMode: 'grid' });
+    render(<LayoutButton isScreenSharePresent={false} isPinnedPresent />);
+    const button = await screen.findByRole('button');
+    await userEvent.hover(button);
+    const tooltip = await screen.findByRole('tooltip');
+    expect(tooltip).toBeInTheDocument();
+    expect(tooltip.textContent).toBe('Cannont switch layout while a participant is pinned');
```

Comment:
Nit: 'Cannont' -> 'Can not' ?

Reply:
ope, typo in both the implementation and the test so that's why the tests were passing 😂 great catch, I'll update to `Cannot`, that's how we currently have it with screen share